pkg/pillar/cmd/zedkube/zedkube.go

```diff
@@ -103,31 +204,173 @@ func Run(ps *pubsub.PubSub, loggerArg *logrus.Logger, logArg *base.LogObject, ar
 	zedkubeCtx.subGlobalConfig = subGlobalConfig
 	subGlobalConfig.Activate()
 
-	err = kubeapi.WaitForKubernetes(agentName, ps, stillRunning)
+	// Watch DNS to learn which ports are used for management.
```

Comment:
This comment is wrong - zedkube subscribes to DNS to find out when the cluster IP address is ready - i.e. configured by NIM.

Reply:
Yeh, copied comments also from domainmgr.go, will modify.